### PR TITLE
[CI] Add integration test for MariaDB 11.4 and Oracle 19 

### PR DESCRIPTION
**Pull request by Torch3333**
## Description

This adds integration tests for MariaDB 11.4 and Oracle 19 in the CI.

## Related issues and/or PRs

N/A

## Changes made

- Add integration test for MariaDB 11.4 and Oracle 19 in the CI

## Checklist

> The following is a best-effort checklist. If any items in this checklist are not applicable to this PR or are dependent on other, unmerged PRs, please still mark the checkboxes after you have read and understood each item.

- [x] I have commented my code, particularly in hard-to-understand areas.
- [ ] I have updated the documentation to reflect the changes.
- [ ] Any remaining open issues linked to this PR are documented and up-to-date (Jira, GitHub, etc.).
- [ ] Tests (unit, integration, etc.) have been added for the changes.
- [ ] My changes generate no new warnings.
- [ ] Any dependent changes in other PRs have been merged and published.

## Additional notes (optional)

While there are completely free Oracle DB for commercial use for versions 18, 21 and 23, no free version for Oracle 19 is available. However, we can use it as long as it is for testing purposes.

## Release notes

Supports MariaDB 11.4 and Oracle 19.

### CODE REVIEW

File: .github/workflows/ci.yaml [hunk 1]
```diff
@@ -952,6 +952,70 @@ jobs:
           name: oracle_18_integration_test_reports_${{ matrix.mode.label }}
           path: core/build/reports/tests/integrationTestJdbc
 
+  integration-test-for-jdbc-oracle-19:
```
Comment (brfrn169) [inline .github/workflows/ci.yaml:955]: Since we've added this, we can remove `integration-test-for-jdbc-oracle-18`.

**Review by brfrn169 — APPROVED** [review]
Left a comment. Other than that, LGTM! Thank you!

File: .github/workflows/ci.yaml (hunk 1 above)
Comment (Torch3333) [reply .github/workflows/ci.yaml:884]: Ok, it's revised in https://github.com/scalar-labs/scalardb/pull/2061/commits/d4c15a05ba3450c35c028249aef12403b9e35ed2

Comment (Torch3333) [reply .github/workflows/ci.yaml:884]: Do we keep Oracle 18 on the CI for already released versions? Do you think about keeping Oracle 18 on the [Supported database](https://scalardb.scalar-labs.com/docs/latest/scalardb-supported-databases/) documentation or replacing it with Oracle 19 ?

Comment (brfrn169) [reply .github/workflows/ci.yaml:884]: I think we can delete Oracle 18 from the supported database documentation. @feeblefakie What do you think?

Comment (feeblefakie) [reply .github/workflows/ci.yaml:884]: @brfrn169 I think so. The updated version is in my PR, so let me handle this.

**Review by feeblefakie — APPROVED** [review]
LGTM! Thank you!

**Review by komamitsu — APPROVED** [review]
LGTM! 👍

**Pull request merged**